exercises/space-age/src/example.c

```diff
@@ -0,0 +1,41 @@
+#include "example.h"
+
+float convert_Earth_Age(const long input)
```

Comment:
How about `convert_earth_age`? This will be consistent with the rest of the exercises.